nbgrader/docs/source/configuration/jupyterhub/jupyterhub_config.py

```diff
@@ -7,3 +7,24 @@
 c.Authenticator.whitelist = [
     "instructor1", "instructor2", "student1", "student2", "student3"
 ]
+
+# Add users here that are allowed to access the formgrader. NOTE: this option
+# strictly adds users. If you want to revoke formgrader access after running
+# JupyterHub, you will need to use the JupyterHub API to remove that user
+# from the formgrader group.
+c.JupyterHub.load_groups = {
+    'formgrader': ['instructor1', 'instructor2']
+}
+
+# This is the key piece which defines how JupyterHub should talk to nbgrader
+c.JupyterHub.services = [
+    {
+        'name': 'formgrader',
```

Comment:
I would name this something that makes it clear you have multiple of these such as `formgrader-course1`